services/iam/spec/requests/users_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
-describe 'users requests', type: :request do
+RSpec.describe 'users requests', type: :request do
+  include LoginSpecHelper
```

Comment:
@alexeylightit I've made use of your login helper here to make sure the specs that I had written pass. I'm unsure how they were passing before but they were.
anyway, please take a look and see if this is the planned usage.

Reply:
we should load /spec/support/**/*.rb on initializing state. Don't know where it has gone